src/main/java/org/isf/accounting/rest/BillController.java

```diff
@@ -369,9 +370,10 @@ public ResponseEntity<List<BillDTO>> getPendingBills(@RequestParam(value="patien
 	 * @throws OHServiceException
 	 */
 	@PostMapping(value = "/bills/search/by/item", produces = MediaType.APPLICATION_JSON_VALUE)
-	public ResponseEntity<List<BillDTO>> searchBills(@RequestParam(value="datefrom") @DateTimeFormat(pattern = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'") @Schema(implementation = String.class) LocalDateTime dateFrom,
-			@RequestParam(value="dateto")@DateTimeFormat(pattern = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'") @Schema(implementation = String.class) LocalDateTime dateTo,
-			@RequestBody BillItemsDTO billItemDTO) throws OHServiceException {
+	public ResponseEntity<List<BillDTO>> searchBills(
```

Comment:
please why we have space here ?

Reply:
it's the formatter doing that (`...line_length" value="160"/>`)